KotlinMMP/androidApp/src/androidTest/java/and/jemison/kotlinmmp/androidapp/MainActivityTest.kt

```diff
@@ -1,90 +1,41 @@
 import org.junit.runner.RunWith
 
-import and.jemison.kotlinmmp.androidapp.testutils.DrawableMatcher
-import android.view.View
-import org.hamcrest.Matcher
-
-
 @RunWith(AndroidJUnit4::class)
 @LargeTest
 class MainActivityTest {
 
     @get:Rule
-    val activityRule = ActivityScenarioRule(MainActivity::class.java)
-
-    private lateinit var viewPager2IdlingResource: ViewPager2IdlingResource
-
-    @Before
-    fun setUp() {
-        activityRule.scenario.onActivity {
-            viewPager2IdlingResource =
-                ViewPager2IdlingResource(
-                    it.findViewById(R.id.mood_options_view_pager),
-                    "viewPagerIdlingResource"
-                )
-            IdlingRegistry.getInstance().register(viewPager2IdlingResource)
-        }
-    }
-
-    @After
-    fun tearDown() {
-        IdlingRegistry.getInstance().unregister(viewPager2IdlingResource)
-    }
+    val composeTestRule = createAndroidComposeRule<MainActivity>()
 
     @Test
-    fun showMoodOptions() {
-        checkGoodMoodDisplayed()
-
-        onView(withId(R.id.mood_options_view_pager)).perform(swipeLeft())
-
-        checkBadMoodDisplayed()
-
+    fun pageRenders() {
+        composeTestRule.onNodeWithTag(MOOD_BACKGROUND).assertIsDisplayed()
+        composeTestRule.onNodeWithText(HOW_ARE_YOU).assertIsDisplayed()
+        composeTestRule.onNodeWithText(SWIPE_HELP).assertIsDisplayed()
 
-        onView(withId(R.id.mood_options_view_pager)).perform(swipeRight())
+        val defaultMoodText = composeTestRule.onNodeWithText(GOOD_MOOD_TEXT)
+        defaultMoodText.assertIsDisplayed()
+        composeTestRule.onNodeWithText(BAD_MOOD_TEXT).assertIsNotDisplayed()
 
-        checkGoodMoodDisplayed()
-    }
-
-    private fun checkGoodMoodDisplayed() {
-        onView(withText(GOOD_MOOD_TEXT)).check(matches(isCompletelyDisplayed()))
-        onView(withText(BAD_MOOD_TEXT)).check(doesNotExist())
-        onView(withId(R.id.moodimage)).check(matches(withDrawable(R.drawable.moodhappy)))
-    }
-
-    private fun checkBadMoodDisplayed() {
-        onView(withText(BAD_MOOD_TEXT)).check(matches(isCompletelyDisplayed()))
-        onView(withText(GOOD_MOOD_TEXT)).check(doesNotExist())
-        onView(withId(R.id.moodimage)).check(matches(withDrawable(R.drawable.moodnothappy)))
+        composeTestRule.onNodeWithText(SUBMIT).assertIsDisplayed()
+        composeTestRule.onNodeWithText(DISCLAIMER).assertIsDisplayed()
     }
 
     companion object {
-        private const val GOOD_MOOD_TEXT = "Doing Great"
+        private const val MOOD_BACKGROUND = "moodBackgroundPurple"
+        private const val HOW_ARE_YOU = "How are you feeling today?"
+        private const val SWIPE_HELP = "(Swipe to change mood)"
+        private const val SUBMIT = "Submit"
+        private const val DISCLAIMER = "This is completely anonymous."
+        private const val GOOD_MOOD_TEXT = "Doing Great!"
         private const val BAD_MOOD_TEXT = "Not So Great"
     }
```

Comment:
Would it make sense to end any test constants with either `_TEXT` or `_TAG` to clarify what it refers to? Then it's clear at a glance whether `onNodeWithText()` or `onNodeWithTag()` is the correct function to use